template.tpl

```diff
@@ -197,9 +203,12 @@ function trackByInsightTag() {
       lintrk('track', options);
     }
     data.gtmOnSuccess();
-  } else {
+  } else if (!isScriptInjected) {
+    isScriptInjected = true;
     setInWindow('_already_called_lintrk', true, true);
     injectScript('https://snap.licdn.com/li.lms-analytics/insight.min.js', didInjectInsightTag, didFailInsightTag);
+  } else {
+    didInjectInsightTag();
```

Comment:
Why is this needed?

Reply:
Yes exactly. so we won't miss the conversion signals.
If we got to this else it means the code above didn't run yet (lintrk is not available) so we trigger this function again.